network/p2p/consumer.go

```diff
@@ -0,0 +1,17 @@
+package p2p
+
+import (
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// NodeBlockListConsumer consumes notifications from the cache.NodeBlocklistWrapper whenever the block list is updated.
+// Implementations must:
+//   - be concurrency safe
+//   - be non-blocking
+type NodeBlockListConsumer interface {
```

Comment:
This is pretty specific. Maybe this should be `IdentityProviderConsumer` so we can add other event types related to identity updates later

Reply:
Lets keep the name NodeBlockListConsumer because we are consuming events from the node block list wrapper the naming is consistent. When dynamic identity table is implemented we will remove this consumer and update the naming.